src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -127,7 +127,7 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
      * @return {Promise}
      * @resolved {SavedObject}
      */
-    this.init = _.once(() => {
+    this.init = _.once((isDefered) => {
```

Comment:
> That said, I think all calls to docSource.fetch() should probably be using the request queue.

What about when just retrieving a single object by id?  E.g. this gets called when you save a dashboard because it reloads the dashboard you just saved by id.  In that case, maybe it's not worth the overhead?  It looks like the looper checks for requests every 1.5 seconds, so that's adding an average .75s delay when just getting a single object.

That said, I agree about the naming, but what about batchRequest? I like `batch` better than `defer`, as it gives more reasoning behind it.

I feel like the entire batching logic could be improved to circumvent the looper delay entirely.  Send in an array of savedObjects, or ids, and grab them all at once in a single batch (or split batches up to avoid any one batch from being too large if that was a potential problem).  But that would be too large of a code overhaul at this point, so IMO, just doing the rename might be the best way to go at this point.